### ISSUE TITLE
Implement custom IPAddressField that stores IP in binary but displays in plain text

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#19622]
In https://github.com/mozilla/addons-server/pull/19616 a `VarBinaryField` was implemented to store IP addresses as a `VARBINARY(16)`, to be compatible with MySQL's `INET6_ATON()` and friends and make lookups fast.

At the moment we still have the plain text column in `IPLog`, but it's redundant. It would be nice if we had a single field that would store the IPs as binary, and display plain text to users automatically.

**Comment by ioanarusiczki** [comment]
@diox I assume the field is not visible for me in admin, but if there's something I could check on, let me know.

**Comment by diox** [comment]
As long as IP searches in the admin still work as before then we're good.

**Comment by ioanarusiczki** [comment]
@diox 

I found a scenario that throws 504 Gateway Time-out every time on -dev -> I did not reproduce it on -stage. 

1. I go to `admin/models/users/userprofile/` and make a query using 2 ips, Results are returned (this works with 10+ ips or more, I did not have issues).
2.  After results are returned, I scrolled down the list until I found a pink IP and added it in the query (with the green +) . I'm noticing the search bar is orange.
3. Search again and wait the results -> 504 

To access again the admin I reload https://addons-internal.dev.mozaws.net/en-US/admin/
